src/earn/prepareTransactions.ts

```diff
@@ -196,7 +196,7 @@ export async function prepareWithdrawTransactions({
   }
 }
 
-export function usePrepareTransactions(mode: EarnEnterMode) {
+export function usePrepareTransactions(mode: EarnActiveAction) {
```

Comment:
the implementation of this function seems wrong now, if `mode` is `exit | claim-rewards` this would end up calling `prepareDepositTransactions`